cpg_workflows/defaults.toml

```diff
@@ -1,207 +0,0 @@
-[workflow]
```

Comment:
This file is currently used as a reference by the analysis teams to help them understand the different params that can be overridden in a config. If we remove it in this PR do you think we should replace it with some other form of documentation?

Reply:
The entire contents of that file is in `__init__.py`. Doesn't that work?